basil/utils/USBBinds.py

```diff
@@ -105,7 +109,10 @@ def find_usb_binds(rm, log,
                     log.debug(f"Found memorized bind {res}")
                     result = memorized_binds[res]
                 else:
-                    result = query_identification(rm, res, instrument['baud_rate'], instrument['read_termination'], instrument['write_termination'], timeout=timeout)
+                    if instrument["type"].lower() == "iseg_hv":
```

Comment:
Maybe make a global "devices_with_non_default_values" dict, so that it is easier to potentially add more devices there instead of blowing up an `if` clause?